NSS-Day-Cohort-42/holiday-road-squirrely-squirrels
# add score rank and filter to eateries

To verify:

1. confirm that eateries are now assigned a score based on their attractiveness to the user profile Anna
1. confirm that eateries are listed in Descending order based on that score
1. confirm that eateries with a score less than 3 are NOT shown.
